experimental/dds/tree/src/interface-prototype/Anchors.ts

```diff
@@ -0,0 +1,99 @@
+import { ConstraintEffect } from '../default-edits';
```

Comment:
From our discussion of ops, I was expecting Anchor to be something like:
```ts
interface Anchor {
    node: NodeId;
    trait: string;
    position: "first" | "before" | "after" | "last"
}
```

Reply:
I'll do some renames and add some comments to try and clarify this.

Reply:
Hopefully its more clear now.

What you are describing is more at the abstraction level of StablePlace in experimental\dds\tree\src\default-edits\PersistedTypes.ts . The purpose of this proposed API is to provide something nicer to work with than those types + snapshots (aka the API you get in Checkout), while capturing enough information to enable replaying Commands as higher levels of amstraction than raw shared tree edits when desired (and the command is available)